src/xcode/ENA/ENA/Source/Views/BaseElements/ENAButton.swift

```diff
@@ -15,55 +15,148 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import Foundation
 import UIKit
 
-/// A Button which has the same behavior of UIButton, but with different tint color.
 @IBDesignable
-class ENAButton: UIButton {
-	override init(frame: CGRect) {
-		super.init(frame: frame)
-		customizeButton()
+class ENAButton: DynamicTypeButton {
```

Comment:
This button was crashing IB. I've just checked it, and it seems it doesn't anymore, but please double check this.

Reply:
It needs a clean build for it work properly on the first attempt, yes. After that it should be resolved. This is due to the way named UIColors are handled.

Reply:
If you encounter another issue, please post your IB log file with the corresponding stack trace.